app/jobs/users_summary_generate_job.rb

```diff
@@ -0,0 +1,18 @@
+class UsersSummaryGenerateJob < ApplicationJob
+  queue_as :default
+
+  def perform(today)
+    users = User.all    
+    csv_data = CSV.generate(encoding: 'sjis') do |csv|
+      csv << ['ユーザー名', '対象年月', '投稿数']
+      users.each do |user|
+        date = Date.strptime(ENV["SERVICE_START_DATE"], '%Y/%m/%d').beginning_of_month
```

Comment:
このために環境変数を増やすよりも、実データから取るほうがよさそうに思いました
```suggestion
        date = (Post.minimum(:created_at) || Time.current).beginning_of_month
```

Reply:
なるほどです、最も古いPostの日付はサービス開始日とみなせそうですね。
修正しました→[29ac710](https://github.com/tomo-ogawa-andpad/my_benkyo_rails_app/pull/1/commits/29ac710882d204402f049674f6966be23c94ea58)

Time.currentはTime.zone.nowと同じなのですね
https://api.rubyonrails.org/classes/Time.html#method-c-current

Reply:
Yes 👍 
コードの内容を深追いするのはよいプラクティスですね